### PR TITLE
fixed startup bug

### PR DESCRIPTION
**Pull request by ThibaultRouillard**
There was a `#KeyError: 'initialized`

Adding the `return` in a `try:` fixes it.

### CODE REVIEW

File: rqt_controller_manager/src/rqt_controller_manager/controller_manager.py [hunk 1]
```diff
@@ -383,7 +383,11 @@ def data(self, index, role):
 
         if role == Qt.DecorationRole:
             if index.column() == 0:
-                return self._icons[ctrl.state]
+                
+                try:                                
+                    return self._icons[ctrl.state]  #KeyError: 'initialized'
+                except:
```
Comment (bmagyar) [inline rqt_controller_manager/src/rqt_controller_manager/controller_manager.py]: shouldn't we specifically catch the `KeyError` you put in the comment?

**Review by bmagyar — CHANGES_REQUESTED** [review]

File: rqt_controller_manager/src/rqt_controller_manager/controller_manager.py (hunk 1 above)
Comment (ThibaultRouillard) [reply rqt_controller_manager/src/rqt_controller_manager/controller_manager.py]: Actually, there is a missing `ctrl.state`: `initialised`

**Comment by wxmerkt** [comment]
This is currently broken in Melodic from binaries so it would be great to get a fix in ASAP :-).

**Comment by bmagyar** [comment]
@ThibaultRouillard @wxmerkt I've cleaned up the PR and it is now ready for merge. As we don't have CI specifically for rqt plugins I have to ask you guys for confirmation that this fixes the issue you are seeing. As soon as this is in I'm ready to push the next melodic release.

**Review by bmagyar — APPROVED** [review]

**Comment by wxmerkt** [comment]
Thank you for the swift action :-) I can confirm this fixes the issue

**Comment by ThibaultRouillard** [comment]
Sorry for the delay. Thank you for accepting the request.

**Comment by bmagyar** [comment]
https://github.com/ros/rosdistro/pull/23860

**Pull request merged**